scripts/toolbox/tests/RocksDBCacheSizeTest.py

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env python
+
+import importlib.util
+
+
+def module_from_file(module_name, file_path):
+    spec = importlib.util.spec_from_file_location(module_name, file_path)
+    module = importlib.util.module_from_spec(spec)
+    spec.loader.exec_module(module)
+    return module
+
+
+Feed = module_from_file("Feed", "modules/Feed.py")
```

Comment:
It is actually easier to import functionality from modules. This is all you need:
```suggestion
from modules import Feed
```

Reply:
Indeed .. awesome. IDE also confused me on this one: 

![image](https://github.com/arangodb/arangodb/assets/1244163/03ee9906-3b5c-46db-9668-ca123220949a)

Reply:
![image](https://github.com/arangodb/arangodb/assets/1244163/ff0268b1-5115-4fb3-acd4-1a1981e06879)

Reply:
^-- Issues has been my local IDE. just as a note.